src/neo/Ledger/MemoryPool.cs

```diff
@@ -416,7 +416,7 @@ internal void InvalidateAllTransactions()
             // Since unverifiedSortedTxPool is ordered in an ascending manner, we take from the end.
             foreach (PoolItem item in unverifiedSortedTxPool.Reverse().Take(count))
             {
-                if (item.Tx.VerifyForEachBlock(snapshot, SendersFeeMonitor.GetSenderFee(item.Tx.Sender)) == RelayResultReason.Succeed)
+                if (item.Tx.Verify(snapshot, SendersFeeMonitor.GetSenderFee(item.Tx.Sender)) == RelayResultReason.Succeed)
```

Comment:
We can keep `VerifyForEachBlock()` to improve the performance of mempool.

Reply:
And move `VerifyParallelParts` to `Verify` ?

Reply:
Yes.

Reply:
Done